backend/src/index.ts

```diff
@@ -4,16 +4,25 @@ import router from "./routes";
 import mongoose from "mongoose";
 import Gateway from "./gateway";
 import { createServer } from "http";
+import rateLimit from "express-rate-limit";
 
 const app = express();
 const PORT: number = parseInt(process.env.PORT as any) || 3000;
 const MONGODB_URI: string = process.env.MONGODB_URI || "mongodb://127.0.0.1:27017/FPSCR";
 mongoose.set('strictQuery', false);
 
+const limiter = rateLimit({
+	windowMs: 15 * 60 * 1000, // 15 minutes
```

Comment:
this seems really stringent, lets make it more relaxed. Every time someone views or comments or likes it is going to be rate limited. Lets make it like 50 every 2 minutes or something,

Reply:
100 requests every 15 minutes is 6-7 requests a minute so that's why I thought it was reasonable but yeah i'll change it